_includes/specification_table.html

```diff
@@ -77,7 +77,7 @@
                 <td>
                     {% for temp_exp_type in prop.expected_types %}
                     {% assign expected_type_type = temp_exp_type | split: ':' %}
-                    {% if temp_exp_type == "LabProtocol" or temp_exp_type == "DataRecord" or temp_exp_type == "BioChemEntity" %}
+                    {% if temp_exp_type == "LabProtocol" or temp_exp_type == "DataRecord" or temp_exp_type == "BioChemEntity" or temp_exp_type == "Gene" or temp_exp_type == "Taxon"%}
```

Comment:
This works for now, but I feel that this is not a very elegant way of doing this.